docs/source/cuda.rst

```diff
@@ -117,6 +117,7 @@ Memory management
      reset_peak_memory_stats
      caching_allocator_alloc
      caching_allocator_delete
+     caching_allocator_enable
```

Comment:
All of these are actually coming from torch.cuda.memory and not torch.cuda directly. That's why the docs coverage script is complaining.
You can change the current module with `.. currentmodule:: torch.cuda.memory` before defining the APIs coming from there to make them properly documented

Reply:
I tried your suggestion of adding `.. currentmodule:: torch.cuda.memory`, but it causes all of the following APIs to report a warning such as
`/var/lib/jenkins/pytorch/docs/source/generated/torch.cuda.MemPool.rst: WARNING: document isn't included in any toctree`

Can I go back to using coverage_ignore_functions?  lol.  it's simpler.  and the coverage report of caching_allocator_enable seems to be a false positive anyway.  The function does show up in the docs anyway.

Reply:
@svekars I hear you're a docs expert.  Any ideas how to get past this docs issue without modifying [docs/source/conf.py](https://github.com/pytorch/pytorch/pull/131114/files/f9e2c1198f54ebeddcb251ae788fa88bb08855d7#diff-008dcb3426febd767787b1521f1fe33086313b927ea37eaab86df5fa88a51698) to skip the failure?